REDACTED-params.yaml

```diff
@@ -66,6 +66,6 @@ concourse:
   namespace: concourse
   fqdn: concourse.dorn.tkg-aws-e2-lab.winterfell.live
   tmc-workspace: concourse-workspace
-argocd:
```

Comment:
Why removing the argocd configuration when adding kubeapps.
It should be fine to keep both options

Reply:
We pulled this to mod a few things, but i deleted Argo because it was in there twice.